src/harness/skeleton.rs

```diff
@@ -270,6 +270,13 @@ pub fn build_run_directory(
     copy_harness_file(&run_to_cp, &run.join(RUN_RUN_FILE))?;
     copy_harness_file(env_file, &run.join(RUN_ENV_FILE))?;
 
+    // add repetition number to env file
+    let mut env_file = OpenOptions::new()
+        .append(true)
+        .open(&run.join(RUN_ENV_FILE))
```

Comment:
See EXP_SRC_DIR PR about the "There should be some struct to represent environments" thing.

This should be one of those "reserved environment variables" that exomat env refuses to add and which are set by exomat after the environments from the file has been set, then we don't need this "Write REP to env file magic"